primitives/core/src/crypto.rs

```diff
@@ -950,7 +950,7 @@ pub trait Pair: CryptoType + Sized + Clone + Send + Sync + 'static {
 	/// Similarly an empty password (ending the SURI with `///`) is perfectly valid and will
 	/// generally be equivalent to no password at all.
 	///
-	/// `None` is returned if no matches are found.
+	/// an error is returned if no matches are found.
```

Comment:
You can probably just remove this line - there are multiple reasons for an error return and I dont see the need to list them all here.

Reply:
Sorry just now seeing this @ggwpez